DigitalOffice/Tests/UserServiceTests/ValidatorsTests/UserCreateRequestValidatorTests.cs

```diff
@@ -4,7 +4,7 @@
 using LT.DigitalOffice.UserService.RestRequests;
 using LT.DigitalOffice.UserService.Validators;
 
-namespace LT.DigitalOffice.UserServiceUnitTests.Validators
+namespace LT.DigitalOffice.UserServiceUnitTests.ValidatorsTests
```

Comment:
It is necessary to change the name from ValidatorsTests to Validators.

Reply:
Fixed